tools/packaging/common/envoy_bootstrap.json

```diff
@@ -96,12 +92,6 @@
         "tag_name": "mongo_prefix",
         "regex": "^mongo\\.(.+?)\\.(collection|cmd|cx_|op_|delays_|decoding_)(.*?)$"
       },
-      {{- range $a, $tag := .extraStatTags }}
```

Comment:
I'd suggest to keep it here, but change the default value to be empty.

Reply:
seems reasonable.